libraries/wasm-jit/Include/IR/Operators.h

```diff
@@ -11,7 +11,8 @@ namespace IR
 
 	struct NoImm {};
 	struct MemoryImm {};
-
```

Comment:
Can you clean up the whitespace differences in wasm-jit directory? These sort of things clutter diffs to upstream.

Reply:
Got it.